runtime/jcl/common/jclcinit.c

```diff
@@ -66,87 +67,85 @@ jint computeFullVersionString(J9JavaVM* vm)
 {
 	VMI_ACCESS_FROM_JAVAVM((JavaVM*)vm);
 	PORT_ACCESS_FROM_JAVAVM(vm);
+	OMRPORT_ACCESS_FROM_J9PORT(vm->portLibrary);
 	const char *osarch;
 	const char *osname;
 	const char *vmVersion = NULL;
 	const char *gcVersion = NULL;
-	char fullversion[512];
 	char vminfo[512];
+        char fullversion[512];
```

Comment:
Please use tabs in all the modified code, to be consistent with the rest of the file.